new/LIT7338TammeraL.xml

```diff
@@ -48,7 +48,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         </revisionDesc>
     </teiHeader>
     <text>
-        <t:body xmlns:t="http://www.tei-c.org/ns/1.0">
+        <body xmlns:t="http://www.tei-c.org/ns/1.0">
```

Comment:
you had to delete the xmlns link as well

Reply:
Ok! I will reopen it and delete!